server/routes/login/index.post.ts

```diff
@@ -51,7 +57,7 @@ export default eventHandler(async (event) => {
       });
     }
 
-    return userDocument;
+    return omit(userDocument.toObject(), ["privateKey"]);
```

Comment:
Use flattenMaps, because meta will not be transformed to object
```suggestion
    return omit(userDocument.toObject({ flattenMaps: true }), ["privateKey"]);
```